cloudformation/cfncluster.cfn.json

```diff
@@ -759,10 +753,9 @@
       "Type": "AWS::EC2::AvailabilityZone::Name"
     },
     "EBSSnapshotId": {
-      "Description": "Id of EBS snapshot if using snapshot as source for volume",
+      "Description": "Comma delimited list of Id of EBS snapshot if using snapshot as source for volume",
       "Type": "String",
-      "Default": "NONE",
-      "AllowedPattern": "(NONE|^snap-[0-9a-z]{8}$|^snap-[0-9a-z]{17}$)"
```

Comment:
We can probably also keep this validation by chaning the regex as  `^(NONE|snap-[0-9a-z]{8}|snap-[0-9a-z]{17})(,(NONE|snap-[0-9a-z]{8}|snap-[0-9a-z]{17}))*$`